Defs/Ammo/Pistols/9x19mm M1938.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Defs>
+	
+	<!-- ========== Based off of 9x19mm Parabellum ========== -->
+	
+	<ThingCategoryDef>
+		<defName>Ammo9x19mmM1938</defName>
+		<label>9x19mm M1938</label>
+		<parent>AmmoPistols</parent>
+		<iconPath>UI/Icons/ThingCategories/CaliberPistol</iconPath>
+	</ThingCategoryDef>
+
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_9x19mmM1938</defName>
+		<label>9x19mm M1938</label>

```

Comment:
From what I've found this is just a hotter version of the above 9mm Gil, making it ever closer to 9mm Para and I think the m1938 could use 9mm Para. Would recommend this just goes away.